### DISCUSSION TITLE
Allow to disable Grafana Managed Alerts

### DISCUSSION THREAD

**Discussion by kubicgruenfeld** [grafana/grafana#65239] [Alerting]
Our users get confused by "Grafana Managed Alerts" since they should add alerts only in Loki and Mimir since our Grafana is not high available, but Loki and Mimir is.

There is even a hint in the UI: Select “Grafana managed” unless you have a Mimir, Loki or Cortex data source with the Ruler API enabled.", but you can not disable it for now.

We would be happy to provide a PR to allow disabling "Grafana Managed Alerts".

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82783. Feel free to continue the discussion around this there. Thank you!
